airflow/dag_processing/manager.py

```diff
@@ -383,14 +383,15 @@ def __init__(
         self._dag_ids = dag_ids
         self._async_mode = async_mode
         self._parsing_start_time: Optional[int] = None
+        self._standalone_mode = conf.getboolean("scheduler", "standalone_dag_processor")
```

Comment:
I think we do not need it here (see below). Using `signal_conn is not None` is more than enough I think and it will simplify quite a few ifs below (and removes the need for mental correlation between `standalone_mode` and `signal_conn`)